console/console-init/ui/src/Components/AddressSpace/AddressSpaceHeader.css

```diff
@@ -0,0 +1,15 @@
+.l_split_m_gutter_MarginRight {
+  margin-right: 10px;
+  font-size:1.5rem;
+}
+.l_split_m_height {
+  font-size: 2.5rem;
+}
+.pf-c-dropdown__menu-item {
```

Comment:
We can't change a `pf` class without namespacing it somehow. Other parts of the app that end up rendering elements with the same `pf` class would be affected by this.